### DISCUSSION TITLE
[Q] What's the text model corresponding to the image tower `datacomp_xl_s13b_b90k`?

### DISCUSSION THREAD

**Discussion by NightMachinery** [mlfoundations/open_clip#534] [Announcements]

**Comment by gabrielilharco** [comment]
Hey @NightMachinery, the config for the text tower can be found at https://huggingface.co/laion/CLIP-ViT-L-14-DataComp.XL-s13B-b90K/blob/main/open_clip_config.json#L10

**Reply by NightMachinery** [reply]
Thanks! But which weights does it actually load? There is no hub ID in the config. Can you give me a reference to where the weights come from? Who released those weights? OpenAI?

**Reply by gabrielilharco** [reply]
We also train the text tower from scratch

**Reply by leng-yue** [reply]
Hi, Is there a training command we can refer to?

**Reply by gabrielilharco** [reply]
There are more details at https://github.com/mlfoundations/datacomp
